tools/bazel_stamp_vars.js

```diff
@@ -43,7 +43,7 @@ console.log(`BUILD_SCM_LOCAL_CHANGES ${LOCAL_CHANGES}`);
 // This will ignore non-version tags which would break unit tests expecting a valid version
 // number in the package headers
 const BUILD_SCM_VERSION_RAW =
-    _exec(`git describe --match [0-9].[0-9].[0-9]* --abbrev=7 --tags HEAD`);
+    _exec(`git describe --match [0-9]*.[0-9]*.[0-9]* --abbrev=7 --tags HEAD`);
```

Comment:
Nit so that it matches with [pre-check](https://github.com/angular/angular/blob/cab0c9babcc2732fd79e08a4e1cf7a12a44cdd74/scripts/release/pre-check#L12)

```suggestion
    _exec(`git describe --match [0-9]+.[0-9]+.[0-9]+ --abbrev=7 --tags HEAD`);
```

Reply:
I could not tell you why, but it seems that the `--match` regex matching doesn't support `+`?

Reply:
Yeah, looks like it. http://man7.org/linux/man-pages/man7/glob.7.html. It's fine with `*` too.